Header header

bool system_good
bool use_enable_key
bool enable_psx_manual
float64 linear_speed
float64 angular_speed

string message